package.json

```diff
@@ -54,7 +54,8 @@
     "tsc": "tsc --incremental",
     "tsc.prod": "tsc",
     "tsc.scripts": "tsc -p scripts/tsconfig.json",
-    "tsc.watch": "tsc --incremental --watch"
+    "tsc.watch": "tsc --incremental --watch",
+    "build.action": "ncc build ./.github/workflows/actions/auto-request-reviews/main.js -o .github/workflows/actions/auto-request-reviews"
```

Comment:
This will go away in final code. Not sure if we need to have some script to do this. Alternatively, we can publish the action outside of the Stencil repo so we wouldn't need to keep any of the action-specific code around

Reply:
Moved this to the action's dedicated `package.json` so we have a quick way of compiling the action if it changes (or deps change)